isort/hooks.py

```diff
@@ -18,7 +18,7 @@ def get_output(command: List[str]) -> str:
     :param str command: the command to run
     :returns: the stdout output of the command
     """
-    result = subprocess.run(command, stdout=subprocess.PIPE, check=True)  # nosec - trusted input
+    result = subprocess.run(command, stdout=subprocess.PIPE, check=True)  # nosec # trusted input
```

Comment:
This one I believe the linter will not complain if you put `# Trusted input` in the line before.